### ISSUE TITLE
log_y=True produces empty plot

### ISSUE THREAD

**Issue by dorgol** [plotly/plotly.py#3297]
I have the following case:

import plotly.express as px
df = pd.DataFrame(np.random.randint(0,100,size=(100, 2)), columns=['year', 'type'])
fig = px.density_heatmap(df, x='year', y='type')
fig.show()

The returned plot is only the layout but with no actual values in it. 

plotly.__version__ = '4.14.3'

**Comment by TR-EIP** [comment]
Yeah, i have the same problem with density_heatmap. Any workaround found?

**Comment by LiamConnors** [comment]
Hi @archmoj, here's a codepen of the issue in Plotly.js https://codepen.io/Liam-Connors/pen/QWoEZEx
